### DISCUSSION TITLE
How can I contribute for a printer shape and look ? (CR10S-Pro)

### DISCUSSION THREAD

**Discussion by DzzD** [supermerill/SuperSlicer#1805] [Q&A] [answered]
As binder clips on the CR10S Pro are fixed, I have make a simple 3d model to use in Printer profile, so users can see if they print onto the binder clips, when the Bed shape is well tuned it become very usefull. (Binder clips have the right size and location)

![image](https://user-images.githubusercontent.com/12198339/140610733-7e5b4c35-d5e4-4548-afad-67251fc9080d.png)

Note that the picture is also usefull as it contain text "Creality" then users can still know when zoomed where is back and front.

![image](https://user-images.githubusercontent.com/12198339/140610790-ffedefcf-b209-42cb-bd47-0a5ffc9352df.png)

**Comment by supermerill** [comment, marked as answer]
You can submit your vendor config modification via a pull request on https://github.com/slic3r/slic3r-profiles/pulls
If you can't, you can share your modification on an issue. 
Note that the pictures (png or svg) need to be of reasonable size (50k or less is ideal, no more than a few hundred ko)

**Comment by DzzD** [comment]
I managed to reduce the picture size from 1Mo to 75Ko... (by reducing number of color to 8, removing useless transparency ...), it become very hard to make it even smaller without reducing the quality too much.

I will try to reduce the picture even more and do a pull request later, in the mean time I join to this post the stl and png for CR10SPro, if someone use it plz let me know.

I know SuperSlicer is a fork from Slic3r but why the pull request must be done in slic3r and not directly somewhere into SuperSlicer repository ?

[CrealityCR10SPro-Bed-Shape.zip](https://github.com/supermerill/SuperSlicer/files/7490578/CrealityCR10SPro-Bed-Shape.zip)

**Reply by supermerill** [reply]
75k is very good.

Because Slic3r may move to the superslicer codebase, and the projects merge. Needs a bit more work to get it through.
